tool/src/main/java/org/wildfly/security/tool/Command.java

```diff
@@ -204,6 +221,28 @@ public void printDuplicatesWarning(CommandLine cmdLine, List<String> duplicatesA
         }
     }
 
+    /**
+     * Prints out a warning message if silentMode is not enabled and adds the warning to the summary
+     * if summaryMode is enabled
```

Comment:
Just to check, the logic for determining if silentMode is not enabled and determining if summaryMode is enabled is handled elsewhere right. (This is just a minor comment, just that the description of the method doesn't match what the method is doing.)

Reply:
Ah, I see now that these methods aren't new. They were just moved.

Reply:
Yeah, I missed these comments. I'll update them both